Sources/Crypto/RSA/RSAKey.swift

```diff
@@ -102,6 +107,33 @@ final class CRSAKey {
         self.pointer = pointer
     }
 
+    /// Creates a new DER encoded `CRSAKey` from data.
+    static func make(from der: Data) throws -> CRSAKey {
+        guard let derDecoded = Data(base64URLEncoded: der) else {
+            throw CryptoError.openssl(identifier: "rsax509", reason: "Certificate decoding failed")
+        }
+
+        guard let x509 = derDecoded.withByteBuffer({ bufferPointer -> UnsafeMutablePointer<X509>? in
+            var base = bufferPointer.baseAddress
+            let count = bufferPointer.count
+            return d2i_X509(nil, &base, count)
+        }) else {
+            throw CryptoError.openssl(identifier: "rsax509", reason: "Key creation from certificate failed")
+        }
+        defer { X509_free(x509) }
+
+        guard let pubkey = X509_get_pubkey(x509) else {
+            throw CryptoError.openssl(identifier: "rsaPkeyNull", reason: "RSA key creation failed")
+        }
+        defer { EVP_PKEY_free(pubkey) }
+
+        guard let rsa = EVP_PKEY_get1_RSA(pubkey) else {
+            throw CryptoError.openssl(identifier: "rsaPkeyGet1", reason: "RSA key creation failed")
```

Comment:
This part of the function is duplicated from the method below. Refactoring the original `make` method to also support DER might be a better solution than copying it and making modifications.

Reply:
You're right!